admin/views/hdf/declaration-form.ejs

```diff
@@ -24,14 +24,20 @@
         <% if (!hdfEligibility) { %>
             <span class="govuk-heading-m">Currently unavailable</span>
             <p class="govuk-body">
-                Access to this section will become available when all pupils have completed the official live check or have a reason for not taking the check.
-                Go to <a href="/pupil-status">pupil status</a> to check these and follow the steps required.
+              You cannot complete the HDF until all pupils have completed the check or have a reason recorded for not taking the check.
+              You can view the status of your pupils and advise on next steps to take on the <a href="/pupil-status">See how many pupils have completed the official check</a> page.
+              Headteachers must submit the headteacher's declaration form once all pupils have completed the check, and by no later than 5pm on Friday 24 June.
             </p>
         <% } else { %>
             <p class="govuk-body govuk-!-font-size-24">
-                Confirm all pupils taking the check completed it according to
-                <a class="govuk-link" href="https://www.gov.uk/government/collections/multiplication-tables-check"
-                   target="_blank">administration guidance</a>. In the absence of the headteacher, a delegated senior
+              Headteachers must ensure that their school fully complies with the statutory arrangements for assessing the multiplication tables check.
+              This form allows the headteacher to confirm:
+              <ul>
+                <li>the official check was correctly administered to all eligible year 4 pupils</li>
+                <li>there was an issue relating to the administration of the check</li>
+              </ul>
+
+              This form should be submitted by no later than 5pm on Friday 24 June. In the absence of the headteacher, a delegated senior
```

Comment:
This end date could also be dynamic.

Reply:
request was to keep these changes as simple as possible, and potentially avoid dynamic/logic based changes.